memcache/memcache_discovery_test.go

```diff
@@ -197,6 +197,7 @@ func startMemcachedServer(t *testing.T, port int, ready chan<- bool, exit <-chan
 	cmd := exec.Command("memcached", "-p", strconv.Itoa(port))
 	t.Logf("starting memcached server with command : %v", cmd)
 	if err := cmd.Start(); err != nil {
+		t.Log(err)
```

Comment:
Kill this with :fire:

Reply:
This is... in a test?  And it's actually quite helpful to know why the test failed here.